src/components/experiment-tracking/accordion/accordion.scss

```diff
@@ -21,7 +21,15 @@ $toggle-width: 1.15em;
   font-weight: 600;
 
   &--medium {
+    cursor: pointer;
```

Comment:
Can we set a color variable on this as well when it's a link? Otherwise, just after selecting it, it turns purple (the browser default for when links have been clicked).

Reply:
hey yup just pushed my code for this, sorry didnt realise i didnt push this change yet